app/src/main/java/org/oppia/app/home/HomeActivityController.kt

```diff
@@ -12,7 +13,7 @@ class HomeActivityController @Inject constructor(private val activity: AppCompat
     activity.setContentView(R.layout.home_activity)
     activity.supportFragmentManager.beginTransaction().add(
       R.id.home_fragment_placeholder,
-      HomeFragment()
+      AudioPlayerFragment()
```

Comment:
I don't think we want to ship this. Maybe add a TODO that this needs to be reverted.

Reply:
Done. reverted, just used for testing purposes